src/app/servicosInterface/dashboard.tecnologia.service.ts

```diff
@@ -0,0 +1,21 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { first, tap } from 'rxjs';
+
+import { DashboardTecnologia } from '../modelosInterface/dashboardTecnologia';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/brunodiascultural/bookshelf_v1/pullRequest?prid=8812417)
```suggestion
  providedIn: "root"
```